content/events/events-content.md

```diff
@@ -2,12 +2,33 @@
 
 <!--The text at the top of this page is pulled from `events-snippet.md` in this folder-->
```

Comment:
This pull request is running into the same major issue as https://github.com/edgi-govdata-archiving/EEW-Website/pull/200 – `events-snippet.md` should have the content that should go before the event tracks and EEW web icon